index.bs

```diff
@@ -73,47 +73,42 @@ spec: webidl
   </h2>
   <p><em>This section is non-normative.</em></p>
   <p>
-    This document's goal is to specify an API that will help developers to
-    handle permissions on the Web platform. Web APIs have different ways to
-    deal with permissions. The [[notifications]] API allows developers to
+    This document specifies an API to query and request permissions to
+    powerful features on the Web platform. Web APIs have different ways to
+    deal with permissions. The [[Notifications]] API allows developers to
     request a permission and check the permission status explicitly. Others
-    might only expose the status to web pages when they try to use the API,
-    like the [[geolocation-API]] which fails if the permission was not
+    expose the status to web pages when they try to use the API,
+    like the [[Geolocation-API]] which fails if the permission was not
     granted without allowing the developer to check beforehand.
   </p>
   <p>
-    Being able to know whether an API call is going to prompt is useful in
-    order to provide a good user experience. Unfortunately, more often than
-    not, those prompts can't be controlled by developers.
+    The Permissions API provides the tools that potentially allow developers to
+    improve the user experience of web applications that require
+    multiple permission requests.
```

Comment:
How about "The Permissions API provides tools for developers to control when permission prompts are shown and to give up permissions that are no longer needed."? "tools that potentially allow" sounds like too much hedging.